packages/serverless-aws-s3/utils.js

```diff
@@ -39,6 +39,10 @@ const bucketCreation = async (s3, Bucket) => {
 const ensureBucket = async (s3, name, debug) => {
     try {
         debug(`Checking if bucket ${name} exists.`);
+        debug(`Checking bucket ${name} for uppercase check.`);
+        if(name[0]>=65 && name[0]<=90){
+            throw Error(`Bucket name must not contain uppercase characters`);
```

Comment:
Ahh, i see. so just to recall we need to check that all the characters are in upperCase or not, correct me if i am wrong.

Reply:
The way I see it, the `easiest` way to do it is not to check for uppercase letters at all, but just do `.toLowerCase()` - always. I guess the question is: `do we want to tell the user that he configured an invalid bucket name?`. If yes, then we can do the check.

Reply:
Thats very legit point @Pavel910 but will it be correct behaviour by changing bucket name to make it bug free, IMO we should be just checking and informing the user about the error.
So that bucket name should be named by user itself

Please confirm the exact behaviour needs to be implemented, @doitadrian you want to addon something ?

Reply:
@ng29 let's warn the user, it's better DX.

Reply:
Sure mate, so just wrapping that we will be checking if buck name contains any upperCase letters then just warn user..